Store/mainStore/store.js

```diff
@@ -0,0 +1,343 @@
+const cardData = [
+  {
+    class: "ramen",
+    name: "안성탕면",
+    tags: [
+      "#달콤",
+      "#짭짤",
+      "#된장맛",
+      "#달콤",
+      "#짭짤",
+      "#된장맛",
+      "#달콤",
+      "#짭짤",
+      "#된장맛",
+    ],
+    img: "../media/ramen_1.jpg",
+  },
+  {
+    class: "ramen",
+    name: "열라면",
+    tags: ["#얼큰", "#화끈", "#해장가능", "#얼큰", "#화끈", "#해장가능"],
+    img: "../media/ramen_2.jpg",
+  },
+  {
+    class: "ramen",
+    name: "짜파게티",
+    tags: ["#짜장", "#촉촉", "#꾸덕", "#짜장", "#촉촉", "#꾸덕"],
+    img: "../media/ramen_3.jpg",
+  },
+  {
+    class: "anju",
+    name: "계란말이",
+    tags: [
+      "#치즈",
+      "#추가",
+      "#천원",
+      "#치즈",
+      "#추가",
+      "#천원",
+      "#치즈",
+      "#추가",
+      "#천원",
+    ],
+    img: "../media/anju_1.jpg",
+  },
+  {
+    class: "anju",
+    name: "소시지",
+    tags: [
+      "#소세지",
+      "#아채볶음",
+      "#소세지",
+      "#아채볶음",
+      "#소세지",
+      "#아채볶음",
+    ],
+    img: "../media/anju_2.jpg",
+  },
+  {
+    class: "anju",
+    name: "어묵탕",
+    tags: ["#뜨끈", "#담백", "#깊은맛", "#뜨끈", "#담백", "#깊은맛"],
+    img: "../media/anju_3.jpg",
+  },
+  {
+    class: "anju",
+    name: "치킨가라아게",
+    tags: [
+      "#바삭",
+      "#촉촉",
+      "#달콤",
+      "#바삭",
+      "#촉촉",
+      "#달콤",
+      "#바삭",
+      "#촉촉",
+      "#달콤",
+    ],
+    img: "../media/anju_4.jpg",
+  },
+  {
+    class: "soju",
+    name: "소주",
+    tags: ["#참이슬", "#새로", "#진로", "#참이슬", "#새로", "#진로"],
+    img: "../media/soju_1.png",
+  },
+  {
+    class: "soju",
+    name: "맥주",
+    tags: [
+      "#카스",
+      "#테라",
+      "#클라우드",
+      "#카스",
+      "#테라",
+      "#클라우드",
+      "#카스",
+      "#테라",
+      "#클라우드",
+    ],
+    img: "../media/soju_2.jpg",
+  },
+  {
+    class: "soju",
+    name: "막걸리",
+    tags: ["#지평", "#느린마을", "#밤", "#지평", "#느린마을", "#밤"],
+    img: "../media/soju_3.png",
+  },
+];
+
+// 추가된 데이터 반영
+function newData() {
+  const cardClass = localStorage.getItem("class");
+  const cardName = localStorage.getItem("name");
+  const cardTags = JSON.parse(localStorage.getItem("tags"));
+  const cardImgSrc = "../media/meme.jpg";
+
+  if (cardTags && cardName) {
+    cardData.push({
+      class: cardClass,
+      name: cardName,
+      tags: cardTags,
+      img: cardImgSrc,
+    });
+  }
+  return cardData;
```

Comment:
요렇게 더 깔끔하게 구조분해할당ㅇ 안되낭?! (아닐수도,,, ㅎㅎㅎ)

```suggestion
 const { class: cardClass, name: cardName, tags: cardTags } = localStorage;
  const cardTags = JSON.parse(cardTags);
  const cardImgSrc = "../media/meme.jpg";

  if (cardTags && cardName) {
    cardData.push({ cardClass, cardName, cardTags, cardImgSrc });
  }

  return cardData;
```